CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Changelog
 
-## 1.3.1 (Unreleased)
+## 1.3.1
 
 ### Maintenance
-* Add timestamping to signed jars.
-* Create the `Janitor` in the `Loader` so that it gets a more logical and consistent `ThreadGroup`.
+* Add timestamping to signed jars. [PR #85](https://github.com/corretto/amazon-corretto-crypto-provider/pull/85)
+* Create the `Janitor` in the `Loader` so that it gets a more logical and consistent `ThreadGroup`. [PR #87](https://github.com/corretto/amazon-corretto-crypto-provider/pull/87)
+* Signed with new JCE signing certificate
```

Comment:
/nit Add PR link for this change to match the above convention used

Reply:
Actually none of these changes correspond to this PR. (This is strictly an organizational one). The certificate change is entirely in our build-fleet and doesn't have an associated PR. (It is tracked through other mechanisms.)

So, it is both correct that the last point (new certificate) has no PR and that this PR has no associated change.